### PR TITLE
Explicitly add maintainers in the README

### PR DESCRIPTION
**Pull request by alexpozzi**
As stated in
https://knplabs.com/en/blog/news-for-our-foss-projects-maintenance ,
we'd like to explicitly indicate who is maintaining the project, as the
KNP Labs organization is looking for maintainers.

Thank you @robbieaverill, I hope you are still interested in being a maintainer of this repository and you are okay with these changes!

### CODE REVIEW

**Comment by robbieaverill** [comment]
I’d prefer not to be the only maintainer, but still happy to help out 😄

File: README.md [hunk 1]
```diff
@@ -101,3 +101,10 @@ $client->setPackagistUrl('https://custom.packagist.site.org');
 ## License
 
 `packagist-api` is licensed under the MIT License - see the LICENSE file for details.
+
+## Maintainers
+
+Please read [this post](https://knplabs.com/en/blog/news-for-our-foss-projects-maintenance) first.
+
+This library is maintained by the following people (alphabetically sorted) :
```
Comment (nicolasmure) [inline README.md]: Maybe also state that we're looking for maintainers, as robbieaverill prefers not to be the only person maintaining this project.

Comment (alexpozzi) [reply README.md]: @robbieaverill, is @nicolasmure proposal fine with you?

Comment (robbieaverill) [reply README.md]: Yes, of course :-)

**Comment by robbieaverill** [comment]
Would you like me to remove the ailing HHVM build from Travis?

**Comment by robbieaverill** [comment]
Never mind, I found #53 :-)

**Comment by nicolasmure** [comment]
Thank you @alexpozzi and @robbieaverill :tada:  :)

**Comment by robbieaverill** [comment]
No problem! Would somebody mind reviewing my open PR? I’d feel a bit strange about self merging a functional change like that

**Pull request merged**